src/components/Help.vue

```diff
@@ -64,6 +64,8 @@ defineExpose({
         Initialize a <code>float</code> buffer with uniform random floats between 0 and 1.
         <h4 class="doc-header"><code>[playground::TIME]</code></h4>
         Gives a <code>float</code> uniform the current time in milliseconds.
+        <h4 class="doc-header"><code>[playground::FRAME_ID]</code></h4>
+        Gives a <code>float</code> uniform the current frame index (starting from 0).
```

Comment:
I suspect supporting integer uniforms for this would be helpful

Reply:
Yeah. For clarity it would be nice if this were an integer. I wasn't sure how to do this in the current code, and am just replicating what we do today for time.